lib/payment_processor/go_cardless/populator.rb

```diff
@@ -38,6 +38,10 @@ def mandate
         @mandate ||= client.mandates.get(complete_redirect_flow.links.mandate)
       end
 
+      def bank_account
```

Comment:
I think it's a bit unfortunate how many third party api requests we end up having to make with GC :/